toponetx/utils/normalization.py

```diff
@@ -1,5 +1,6 @@
-"""Normalize of adjacency, boundary matrices of complexes."""
+"""Normalize of Laplacians, (co)adjacency, boundary matrices of complexes."""
 
+from typing import Tuple, Union
 import numpy as np
 import scipy.sparse.linalg as spl
 from numpy import ndarray
```

Comment:
Please don't import `ndarray` like this. It is standard to import `numpy as np` and access everything through that import, i.e., `np.ndarray`.